content/docs/hooks-overview.md

```diff
@@ -201,9 +201,9 @@ function useFriendStatus(friendID) {
 }
 ```
 
-It takes `friendID` as an argument, and returns whether our friend is online.
+यह एक तर्क के रूप में `friendID` लेता है, और लौटता है कि क्या हमारा दोस्त ऑनलाइन है। यह एक आर्गुमेंट के रूप में `friendID` लेता है, और लौटता है कि क्या हमारा दोस्त ऑनलाइन है।
```

Comment:
`यह एक आर्गुमेंट के रूप में `friendID` लेता है, और लौटता है कि क्या हमारा दोस्त ऑनलाइन है।` 

this is added twice. remove one copy of it